lib/internal/bootstrap/web/exposed-window-or-worker.js

```diff
@@ -63,8 +64,9 @@ exposeLazyInterfaces(globalThis, 'perf_hooks', [
 defineReplaceableLazyAttribute(globalThis, 'perf_hooks', ['performance']);
 
 // https://w3c.github.io/FileAPI/#creating-revoking
-const { installObjectURLMethods } = require('internal/url');
+const { installObjectURLMethods, URLPattern } = require('internal/url');
 installObjectURLMethods();
+exposeInterface(globalThis, 'URLPattern', URLPattern);
```

Comment:
I see most interfaces are exposed via exposeLazyInterface helper, should I use this here too?

Reply:
No need. URLPattern is a C++ only implementation and does not have any boot impacts.